generate.ts

```diff
@@ -732,13 +732,38 @@ function templatizeTree(locals: { [key: string]: any }, sourcePath: string, dest
     for (var i = 0; i < filesInDir.length; i++) {
         var filename = filesInDir[i];
         var file = sourcePath + "/" + filename;
+
         if (fs.lstatSync(file).isDirectory()) {
-            if (excludeFolders != null && excludeFolders.includes(filename))
+            var folderExcluded = false;
+            if(excludeFolders != null)
+            {
+                for(var excludedFolderIndex = 0; excludedFolderIndex < excludeFolders.length; excludedFolderIndex++)
+                {
+                    if(excludeFolders[excludedFolderIndex] == filename)
+                    {
+                        folderExcluded = true;
+                        break;
+                    }
+                }
+            }
+            if (excludeFolders != null && folderExcluded)
```

Comment:
nit: This first clause (excludeFolders != null) can be removed. It's checked above.